src/tcache.c

```diff
@@ -1268,6 +1263,8 @@ tcache_boot(tsdn_t *tsdn, base_t *base) {
 	assert(global_do_not_change_tcache_maxclass <= TCACHE_MAXCLASS_LIMIT);
 	global_do_not_change_tcache_nbins =
 	    sz_size2index(global_do_not_change_tcache_maxclass) + 1;
+	/* Pre-compute default bin info. */
+	tcache_bin_info_compute(opt_tcache_ncached_max);
```

Comment:
Use the getter

Reply:
The getter should provide a const pointer for common usage while this array is meant to be modified (initialized) here. It shouldn't use getter.

Reply:
I see. Worth adding the comments to explain this.